database/datasource.js

```diff
@@ -60,12 +60,16 @@ const create = async (modelName, data) => {
 
 const update = async (modelName, data, options = {}) => {
   const model = getModelDefinitions()[modelName];
-  return await model.update(data, options);
+  return await model.update(data, 
+    {where: {id: options}});
 }
 
+//Delete entry by ID
 const destroy = async (modelName, options = {}) => {
   const model = getModelDefinitions()[modelName];
-  return await model.destroy(options);
+  return await model.destroy({
```

Comment:
I would apply the same reasoning here as with the comment on the update function.